CHANGELOG.md

```diff
@@ -54,28 +63,62 @@ All notable changes to this project will be documented in this file.
 - Tenancy Roles for RBAC
 - Self Service UI is enabled and included in build
 
+### Event Switchboard
+- Initiate event processing through Automate
+- Users can add automate handlers to customize event processing
+- Centralized event mappings
+- Moves provider event mappings from appliance file (config/event_handling.tmpl.yml) into the automate model
+- Organization of Events through automate namespaces
+- Event handling changes without appliance restarts
+- Notes:
+  - New events must be added to automate model
+  - Built-in event handlers added for performance
+  - Requires update of the ManageIQ automate domain
  
-
 ### Tenancy
-- Model - new Tenant model associations
- - Automate domains
- - Service Catalogs
- - Catalog Items
-- Assign existing records to root tenant Provider, Automate Domain, Group, TenantQuota, Vm
+- Model 
+  - new Tenant model associations
+  - Automate domains
+  - Service Catalogs
+  - Catalog Items
+  - Assign default groups to tenants
+  - Assign groups to all VMs and Services
+  - Assign existing records to root tenant Provider, Automate Domain, Group, TenantQuota, Vm
+  - Expose VM/Templates association on Tenant model
 - New Automate Service Models
- - Tenant
- - TenantQuota
+  - Tenant
+  - TenantQuota
 - UI
- - RBAC and Roles - Access Roles features exposed
- - New roles created for RBAC
- - Quota Management
+  - RBAC and Roles - Access Roles features exposed
+  - New roles created for RBAC
+  - Quota Management
 - Associate Tenant to Requests and Services
 - Update of VM tenant when owning group changes
-- Tagging support 
+- Tagging support
+- Automate Tenant Quotas
+  - Customizable Automate State Machine for validating quotas for Service, Infrastructure, and Cloud 
+  - Default Setting based on Tenant Quota Model
+  - Backward compatibility
```

Comment:
Sorry, must have put that one in the wrong spot.  I'll fix it after any
other comments.

On Tue, Nov 3, 2015 at 11:04 AM, Jason Frey <notifications@github.com>
wrote:

> In CHANGELOG.md
> <https://github.com/ManageIQ/manageiq/pull/5254#discussion_r43766628>:
>
> >  - Associate Tenant to Requests and Services
> >  - Update of VM tenant when owning group changes
> > -- Tagging support
> > +- Tagging support
> > +- Automate Tenant Quotas
> > +  - Customizable Automate State Machine for validating quotas for Service, Infrastructure, and Cloud
> > +  - Default Setting based on Tenant Quota Model
> > +  - Backward compatibility
>
> What does this mean?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ManageIQ/manageiq/pull/5254/files#r43766628>.
>



-- 
Marianne Feifer
Senior Technical Project Manager
CloudForms Management Engine
mfeifer@redhat.com
irc: mfeifer
freenode: marigold